src/.main/main.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
Looks like this file got committed by mistake?

Reply:
yep, sorry about that.  I'll get rid of it